stardist/models/model2d.py

```diff
@@ -323,9 +379,17 @@ def simple_augmenter(x,y):
         if steps_per_epoch is None:
             steps_per_epoch = self.config.train_steps_per_epoch
 
+        if self._is_multiclass() and classes is None:
+            warnings.warn("Ignoring given classes as n_classes is set to None")
```

Comment:
Don't understand  (`self._is_multiclass()` is True => n_classes is NOT set to None)

Reply:
What does classes set to None even mean?